📝(doc) complete contributing policy (#871)
suitenumerique/docs: CONTRIBUTING.md

Thank you for taking the time to contribute! Please follow these guidelines to ensure a smooth and productive workflow. 🚀🚀🚀

To get started with the project, please refer to the [README.md](https://github.com/suitenumerique/docs/blob/main/README.md) for detailed instructions on how to run Docs locally.

> Contributors are required to sign off their commits with `git commit --signoff`: this confirms that they have read and accepted the [Developer's Certificate of Origin 1.1](https://developercertificate.org/). For security reasons we also requiring [signing your commits with your SSH or PGP key](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).

Reviewer: ```suggestion
Contributors are required to sign off their commits with `git commit --signoff`: this confirms that they have read and accepted the [Developer's Certificate of Origin 1.1](https://developercertificate.org/). For security reasons we also require [signing your commits with your SSH or GPG key](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification).
```

Please also check out our [dev handbook](https://suitenumerique.gitbook.io/handbook) to learn our best practices.

## Help us with translations

You can help us with translations on [Crowdin](https://crowdin.com/project/lasuite-docs).

> Your language is not there? Request it on our Crowdin page 😊 or ping us on [Matrix](https://matrix.to/#/#docs-official:matrix.org) and let us know if you can help with translations and proofreading.

Reviewer: ```suggestion
Your language is not there? Request it on our Crowdin page 😊 or ping us on [Matrix](https://matrix.to/#/#docs-official:matrix.org) and let us know if you can help with translations and/or proofreading.
```

It is nice to add information about the purpose of the pull request to help reviewers understand the context and intent of the changes. If you can, add some pictures or a small video to show the changes.

### Don't forget to: 

> - sign-off your commits

Reviewer: ```suggestion
- signoff your commits
```

> - sign your commits with your key (SSH, PGP etc.)

Reviewer: ```suggestion
- sign your commits with your key (SSH, GPG etc.)
```
